src/notifications/notifications.service.ts

```
@@ -7,33 +12,95 @@ export interface NotificationPayload {
+      return {
+        data: notifications,
+        meta: {
+          total,
+          page,
+          limit,
+          totalPages,
+          hasNextPage,
+          hasPreviousPage,
+        },
+      };
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)